src/infrastructure/httpServer/common/Envelope.ts

```diff
@@ -1,19 +1,19 @@
 import { ConnectorMode } from "../../../ConnectorMode";
```

Comment:
Kann raus? Ich verstehe nicht mehr ganz das ConnectorMode enum bzw. jetzt type. Das wird ja eigentlich nur noch bei der Rückgabe benutzt?

Reply:
Es wird in der Methode als Parameter genutzt. Wenn es raus könnte hätte vscode das schon gelöscht.